packages/vm/benchmarks/run.ts

```diff
@@ -0,0 +1,52 @@
+import Benchmark = require('benchmark')
+import { mainnetBlocks } from './mainnetBlocks'
+
+const onCycle = (event: any) => {
+  console.log(String(event.target))
+}
+
+async function main() {
+  const args = process.argv
+  let suite = null
+  // Choose between benchmarking or profiling (with 0x)
+  if (args[2] === 'benchmarks') {
+    console.log('Benchmarking started...')
+    suite = new Benchmark.Suite()
+  } else {
+    console.log('Profiling started...')
+  }
+
+  const commandUsageStrg =
+    'Usage: npm run benchmarks|profiling -- BENCHMARK_NAME[:NUM_SAMPLES][,BENCHMARK_NAME[:NUM_SAMPLES]]'
```

Comment:
this is nice, but i don't see it being output anywhere

Reply:
Have corrected (next push).